app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1420,6 +1442,7 @@ private void modifyMenu(boolean show) {
         menu.findItem(R.id.down_angle).setVisible(show);
     }
 
+    @RequiresApi(api = Build.VERSION_CODES.JELLY_BEAN)
```

Comment:
@the-dagger Actually i am using finishAffinity() due to which I required a minimum of 16 SDK version.